src/query/server/server.go

```diff
@@ -148,7 +148,7 @@ func Run(runOpts RunOptions) {
 	defer traceCloser.Close()
 
 	if _, ok := tracer.(opentracing.NoopTracer); ok {
-		logger.Info("tracing disabled; set `tracing.backend` to enable")
+		logger.Info("tracing disabled for m3query; set `tracing.backend` to enable")
```

Comment:
hm looks like this tracer creation logic can be shared between m3dbnode and query. mind refactoring

Reply:
mind if i do this in a follow up pr? i still wanna remove the opentracing stuff that is in query anyway (cause now it's in src/x)

Reply:
sure that works